plugins/GSdx/stdafx.cpp

```diff
@@ -71,16 +71,53 @@ void vmfree(void* ptr, size_t size)
 	VirtualFree(ptr, 0, MEM_RELEASE);
 }
 
+static HANDLE s_fh = NULL;
+static uint8* s_Next[8];
+
 void* fifo_alloc(size_t size, size_t repeat)
 {
-	// FIXME check linux code
-	return vmalloc(size * repeat, false);
+	ASSERT(fh == NULL);
+
+	if (repeat >= countof(s_Next))
+		fprintf(stderr, "Memory mapping overflow (%zu >= %u)\n", repeat, countof(s_Next));
+
+	s_fh = CreateFileMapping(INVALID_HANDLE_VALUE, nullptr, PAGE_READWRITE, 0, size * repeat, nullptr);
+	DWORD errorID = ::GetLastError();
+	if (s_fh == NULL)
```

Comment:
If it fails at this stage, the rest of the code in this function won't work as well (though I note that the non-Windows version isn't all that robust either).